app/__init__.py

```diff
@@ -14,30 +18,50 @@
 search_client = SearchClient.create(Config.ALGOLIA_APP_ID, Config.ALGOLIA_API_KEY)
 index = search_client.init_index(Config.INDEX_NAME)
 
+app = Flask(__name__, static_folder='app/static')
+app.config.from_object(Config)
+app.url_map.strict_slashes = False
 
-def create_app(config_class=Config):
-    app = Flask(__name__, static_folder=None)
-    app.config.from_object(config_class)
-    app.url_map.strict_slashes = False
+limiter = Limiter(
+    app,
+    key_func=get_remote_address,
+    default_limits=["200 per day", "50 per hour"]
+)
 
-    Limiter(
-        app,
-        key_func=get_remote_address,
-        default_limits=["200 per day", "50 per hour"]
-    )
+db.init_app(app)
+migrate.init_app(app, db)
 
-    db.init_app(app)
-    migrate.init_app(app, db)
+from app.api import bp as api_bp # noqa
+app.register_blueprint(api_bp, url_prefix='/api/v1')
 
-    from app.api import bp as api_bp
-    app.register_blueprint(api_bp, url_prefix='/api/v1')
+from app.views import bp as view_bp # noqa
```

Comment:
I'm sure it's necessary, but why do we need the `# noqa` here? What's the rule that it's breaking that we need to ignore? Can we fix it somehow rather than ignoring it?

Totally not a blocker, but I want to understand because I feel like it might be adding tech debt

Reply:
flake8 complains that all imports aren't at the top of the file, but when I moved them it broke everything because these imports need to be after the definition of `db`

Reply:
Ok that's fine we can leave it in. Thanks for clarifying